### ISSUE TITLE
Redis consumer has very high resource usage

### ISSUE THREAD

**Issue by JunkMyFunk** [php-enqueue/enqueue-dev#191]
I have configured the Symfony bundle to use Redis via php-redis. The queue producer and consumer are working, however when I run the console command `php app/console enqueue:consume --setup-broker` it seems to poll Redis every few milliseconds causing the PHP script to have about 50% CPU usage and Redis about 20%.

Is there an option to set a timeout to reduce the polling rate and reduce resource usage?

```
PHP 5.6
Redis server 2.8.4
symfony/symfony: 2.8.27
enqueue/async-event-dispatcher: 0.7.5 
enqueue/enqueue: 0.7.9
enqueue/enqueue-bundle: 0.7.8
enqueue/redis: 0.7.7
```

**Comment by makasim** [comment]
There is no such option yet. Here's a solution we made for dbal transport. It might work for redis too.


https://github.com/php-enqueue/enqueue-dev/blob/master/pkg/dbal/DbalConsumer.php#L96

**Comment by makasim** [comment]
The other solution: Add the third argument to queue consumer which is idle time. https://github.com/php-enqueue/enqueue-dev/blob/master/pkg/enqueue-bundle/Resources/config/client.yml#L82

**Comment by makasim** [comment]
There is `--receive-timeout` option is available since 0.7.11. You can use it to increase time a consumer waits on a socket for a message.
